src/tools/rbd/action/Status.cc

```diff
@@ -148,6 +151,10 @@ static int do_show_status(librados::IoCtx& io_ctx, const std::string &image_name
     }
     if (!image_cache_str.empty()) {
       f->dump_string("image_cache_state", image_cache_str);
+      /*
+      auto cache_type = cache_state["cache_type"].get_str();
+      f->dump_string("cache_type", cache_type);
```

Comment:
I don't know how to do it. Is it like this? get string, then dump_string? If so, I would put the part that gets the value into the public code area above.